dbt/include/snowflake/macros/materializations/incremental.sql

```diff
@@ -20,11 +23,27 @@
        Otherwise, play it safe by using a temporary table.
   #} */
 
+  {% if language == "python" and tmp_relation_type != "None" %}
```

Comment:
I didn't check, but should this be:
```suggestion
  {% if language == "python" and tmp_relation_type is not none %}
```

Reply:
very likley swapping between jinja language and everything else is fun haha. change made